search-app/docs/api.md

```diff
@@ -272,8 +272,8 @@ Here is a list of supported extensions and their corresponding MimeTypes:
   * `opendata`  "application/opendata+json" (only supported for collections)
   * `kml`       "application/vnd.google-earth.kml+xml"
   * `native`    "application/metadata+xml" (Returns search results in their individual native formats)
-  * `umm_json`   "application/vnd.nasa.cmr.legacy_umm_results+json" (only supported for collections)
```

Comment:
We do still have umm-json as an extension which is what this is actually documenting (just with the wrong extension - should be a dash instead of underscore). I think we still should leave this documentation in with umm-json until we've actually removed it.

So document umm-json and umm_json.

Reply:
I added the umm-json extension to the list of supported formats.